examples/envs/README.md

```diff
@@ -0,0 +1,23 @@
+---
+runme:
+  id: 01HW1677G0SDVNNYEDJS3PBKS4
+  version: v3
+---
+
+```sh {"id":"01HW167A4SWNJ0AQZGNZ1Z38XZ","name":"vars"}
+export VAR_NAME="Placeholder 1"
```

Comment:
fyi, this is being treated like a `MESSAGE` not a `PLACEHOLDER` right now. I assume that's because the prompt mode is not accounted for yet (code further below). Just calling it out.

- Message uses no quotes
- Placeholder uses single or double quotes